index.js

```diff
@@ -686,6 +688,10 @@ Tree.prototype.select = function (id, opt) {
     }
 
     this._onSelect(d, null, null, opt)
+  } else {
```

Comment:
If I'm reading this correctly, `tree.select(someIdThatDoesNotExist)` would clear out the selection just like `tree.select(null)`. Is that intentional? I _think_ that makes sense, but I point it out because I originally assumed that trying to select a node that doesn't exist would just have no effect.

Reply:
@jerryorr That's correct.